docs/FAQ.md

```diff
@@ -0,0 +1,72 @@
+---
+sidebar_label: "FAQ"
+sidebar_position: 13
+---
+
+## Smart accounts and SDK
+
+**Will I get the same smart account address on all chains?**
+
+Yes, you will get the same address on all chains as long as the owner's address is the same.
+    
+**How can I create another smart account using the same EOA on the same chain?**
+    
+you can pass the `index` in the config while creating the account as mentioned in the createSmartAccountClient [method](https://docs.biconomy.io/Account/methods#createsmartaccountclient). First smart account gets created with index 0 by default. 
+    
+**Where can I find more examples of SDK integrations?**
+    
+https://github.com/bcnmy/sdk-examples
+    
+**How can I increase the timeout in the waitForTransactionHash function?**
+    
+Initialize the bundler with updated configuration, to increase the timeout (milliseconds).
+
+```jsx
+import { createSmartAccountClient, createBundler } from "@biconomy/account";
+
+const biconomySmartAccount = await createSmartAccountClient({
+    signer,
+    bundler: await createBundler({
+    bundlerUrl: "bundler url",
+    userOpReceiptMaxDurationIntervals: { 80001: 60000 },
+    }),
+});
+```
+    
+**Which social login methods are supported?**
+    
+Explore services such as Web3Auth, Particle Network, Magic.link, or Lit Protocol to link a user's social identity with a cryptographic key. This key can then serve as the owner of the Smart Account. Be sure to check out various [signers](https://docs.biconomy.io/Account/signers/) integrations for this purpose.
+    
+**How do I enable session keys for users?**
+    
+Utilize the session key validation [module](https://docs.biconomy.io/Modules/sessionvalidationmodule) 
+    
+## Bundler & Paymaster
+
+**Is the bundler URL the same for all chains?**
+    
+Yes, the bundler URL is same for all testnets. For mainnet you can reach out to us.
```

Comment:
It's not exactly the same... The chainId changes. Could provide the example here?